src/scripts/main.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
```

Comment:
the scroll is still hidden even after menu is closed, please research this issue 
![image](https://user-images.githubusercontent.com/57598825/148759203-140f8349-6987-43ce-afee-a8eb6ab7a58d.png)